include/odp/api/spec/classification.h

```diff
@@ -204,6 +226,12 @@ typedef struct odp_cls_capability_t {
 
 	/** Supported threshold type for BP */
 	odp_threshold_types_t threshold_bp;
+
+	/** Maximum number of mark id value.
+	 * Value zero represents the mark based CoS is not supported.
```

Comment:
Something like this would sound better to me:
"Zero value means that CoS marking is not supported"

Reply:
Ack